packages/vue/src/components/atoms/SfCircleIcon/SfCircleIcon.js

```diff
@@ -21,7 +21,13 @@ export default {
     sizeIcon: {
       type: String,
       default: "100%"
-    }
+    },
+    /**
+     * Sets aria-label for button icon
+     */
+    alt: {
+      type: String,
```

Comment:
```suggestion
      type: String,
      default: ""
```
please set always default value :)

Reply:
No problem, added and updated

Reply:
@tohanian @przspa still need `role`. What if the icon is something unreadable like "home-create-blah-blah" or even worse, a string of SVGPath? Please make `alt` as `required rather than put empty string as default.